src/interface/src/app/home/share-plan-dialog/share-plan-dialog.component.html

```diff
@@ -63,37 +64,47 @@ <h2>Share {{ data.name }}</h2>
         name="questionText"></textarea>
     </mat-form-field>
   </form>
-  <section class="invited-users-list" *ngIf="!showMessageBox">
-    <h3>People with access</h3>
-    <div class="invited-user">
-      <div class="name">Your Name <span>(You)</span></div>
-      <div>Owner</div>
-    </div>
-    <div *ngFor="let invite of invites$ | async" class="invited-user">
-      <div class="name">{{ invite.name }}</div>
-      <button mat-button [matMenuTriggerFor]="menu">
-        {{ invite.role }} <mat-icon>arrow_drop_down </mat-icon>
-      </button>
-      <mat-menu #menu="matMenu" class="list-menu" xPosition="before">
-        <button
-          mat-menu-item
-          *ngFor="let role of roles"
-          class="menu-roles"
-          (click)="changeRole(invite)">
-          <span class="spacer">
-            <mat-icon *ngIf="role === invite.role">check_small</mat-icon>
-          </span>
-          {{ role }}
-        </button>
-        <button mat-menu-item class="menu-divider" (click)="resendCode(invite)">
-          Resend Code
-        </button>
-        <button mat-menu-item (click)="removeAccess(invite)">
-          Remove Access
+  <app-section-loader
+    [isLoading]="isLoading"
+    [hasData]="true"
+    [ngClass]="{ loading: isLoading }">
+    <section class="invited-users-list" *ngIf="!showMessageBox">
+      <h3>People with access</h3>
+      <div class="invited-user">
+        <div class="name">Your Name <span>(You)</span></div>
```

Comment:
Should this stay "Your Name", or do we want to display the user name?

Reply:
🤦 good catch :)